File: bigdata/aws/glue/etl_process/src/aws_glue.py
from com.spc.process.impl.etl_process import ETLProcess
from com.spc.spark.config.config_enum import ConfigEnum

def glue():
    """
    Realiza as configurações para a execução do processo no AWS Glue
    """
    ETLProcess(ConfigEnum.AWS_GLUE.value).main_process()


glue()

File: bigdata/aws/glue/scala_lab/src/main/scala/AWSGlue.scala
import com.spc.process.IProcess
import com.spc.process.impl.MainProcess
import com.spc.spark.config.ConfigEnum

object AWSGlue {
    def main(sysArgs: Array[String]): Unit = {
        val config = ConfigEnum.getInstance(ConfigEnum.AWS_GLUE)
        val spark = config.initSpark()
        val bucketName = config.getbucketName(sysArgs)
        val process: IProcess = new MainProcess()
        process.execute(spark, bucketName)
    }
}